indexing-service/src/main/java/io/druid/indexing/common/task/ArchiveTask.java

```diff
@@ -36,6 +36,7 @@
 public class ArchiveTask extends AbstractFixedIntervalTask
 {
   private static final Logger log = new Logger(ArchiveTask.class);
+  private static final Priority defaultPriority = Priority.LOW;
```

Comment:
I don't think tasks should have any priority set by default, and instead let the user decide.